### ISSUE TITLE
Way to accessint client documents

### ISSUE THREAD

**Issue by fparrar** [Dolibarr/dolibarr#24191]
### Feature Request

What a weird layout dolibarr has with customer documents. It is possible to attach a doc to a task, a doc to a project, a doc to a client. So far so good. But I don't understand why the display of the files hasn't been made to go from more to less as I 'descend' in specificity. In other words, from 'customer documents' you can see all the documents. That from 'project documents' you can see the project documents and the documents of its tasks (but not the documents of 'customer' or 'other projects'). That from 'task documents' you can see the documents only of the task.

### Use case

In other words, from 'customer documents' you can see all the documents. From 'project documents' you can see the project documents and the documents of its contained tasks (but not the documents of 'customer' or 'other projects'documents). From 'task documents' you can see the documents only of the task.
